platforms/corda-ent/charts/notary/values.yaml

```diff
@@ -85,8 +85,12 @@ rpcSettingsUseSsl: false
 networkServices:
   # Ex. doormanURL: "http://idman-internal.test-kd:10000"
   doormanURL:
+  idmanName:
```

Comment:
Could structure this as such, both are fine
```
  networkServices:
    idman:
      url:
      name: 
      domain:   
    networkmap:
      url:
      name: 
      domain:
```

Reply:
These are to be fixed by whoever is doing the notary refactoring